pymemcache/client/base.py

```diff
@@ -684,7 +650,7 @@ def _fetch_cmd(self, name, keys, expect_cas):
                     result[key] = value
                 else:
                     raise MemcacheUnknownError(line[:32])
-        except Exception:
+        except Exception as e:
```

Comment:
Oops, added that during a debugging session, wasn't supposed to be included in the commit